README.md

```diff
@@ -270,20 +210,15 @@ Nu is in heavy development and will naturally change as it matures and people us
 
 Please submit an issue or PR to be added to this list.
 
-###   Integrations
```

Comment:
Why are we deleting the integrations section?

Reply:
All the info's still there under the "Officially Supported By" header; I just thought it was unnecessary to have both "Officially Supported By" and "Integrations" headers.